specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -998,54 +764,239 @@
+      "properties": {
+        "value": {
+          "description": "The collection of roles.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Role"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of roles.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "SchemaField": {
       "allOf": [
         {
-          "$ref": "#/definitions/Entity"
+          "$ref": "#/definitions/NamedEntity"
         },
         {
           "type": "object",
           "properties": {
-            "falseName": {
-              "description": "The string representation of a false value.",
+            "schema": {
+              "description": "The schema of this field."
+            },
+            "displayUnit": {
+              "description": "The display name of the unit of this field.",
               "type": "string"
             },
-            "trueName": {
-              "description": "The string representation of a true value.",
+            "unit": {
+              "description": "The unit of this field.",
               "type": "string"
             }
-          }
+          },
+          "required": [
+            "schema"
+          ]
+        }
+      ]
+    },
+    "ServiceBusQueueEndpoint": {
+      "x-ms-discriminator-value": "ServiceBusQueueEndpoint",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Endpoint"
+        },
+        {}
+      ]
+    },
+    "ServiceBusTopicEndpoint": {
+      "x-ms-discriminator-value": "ServiceBusTopicEndpoint",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Endpoint"
+        },
+        {}
+      ]
+    },
+    "SolutionModel": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Entity"
         },
         {
           "type": "object",
           "properties": {
-            "hideTime": {
-              "description": "Specifies that the time portion of the date-time value should be hidden.",
-              "type": "boolean"
+            "cloudProperties": {
+              "description": "The cloud properties defined by the solution model.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Capability"
+              }
+            },
+            "initialValues": {
+              "description": "The initial values defined by the solution model.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/InitialValue"
+              }
+            },
+            "overrides": {
+              "description": "The overrides defined by the solution model.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Override"
+              }
             }
           }
+        }
+      ]
+    },
+    "StorageEndpoint": {
+      "x-ms-discriminator-value": "StorageEndpoint",
```

Comment:
You don't need this discriminator, since this is the same string that the class name